lib/puppet_x/puppetlabs/cisco_ios/utility.rb

```diff
@@ -290,6 +291,15 @@ def self.convert_no_to_boolean(value)
       return_value
     end
 
+    # Returns Boolean true if given a non-nil value
+    def self.convert_to_boolean(output)
+      if output
```

Comment:
this could be changed to `output.nil? ? false : true`

Reply:
Done